crates/forge/src/lib.rs

```diff
@@ -552,7 +559,6 @@ fn run_with_fuzzing(
         let mut tasks = FuturesUnordered::new();
         // Pattern in order to waiting for things to finish shutting down
         // https://tokio.rs/tokio/topics/shutdown
```

Comment:
This comments should be moved/copied as well.

Reply:
remove It use other pattern